app/src/main/java/org/fossasia/phimpme/gallery/activities/SingleMediaActivity.java

```diff
@@ -542,14 +542,14 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
                             handleEditorImage(data);
                             if (ContentHelper.copyFile(getApplicationContext(), new File(imageUri.getPath()), new File(getAlbum().getPath()))) {
                                 //((ImageFragment) adapter.getRegisteredFragment(getAlbum().getCurrentMediaIndex())).displayMedia(true);
-                                SnackBarHandler.showWithBottomMargin(parentView, getString(R.string.new_file_created), bottomBar.getHeight());
```

Comment:
What was the error in this method, are we getting the 
`bottomBar.getHeight()` some  positive value?. Please check it out !